dockstore-webservice/src/main/java/io/dockstore/webservice/core/BioWorkflow.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *    Copyright 2019 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.dockstore.webservice.core;
+
+import javax.persistence.Column;
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+/**
+ * These represent actual workflows in terms of CWL, WDL, and other bioinformatics workflows
+ */
+@ApiModel(value = "BioWorkflow", description = "This describes one workflow in the dockstore")
+@Entity
+@DiscriminatorValue("bioworkflow")
+@Table(name = "workflow")
+@SuppressWarnings("checkstyle:magicnumber")
+public class BioWorkflow extends Workflow {
```

Comment:
I'm hoping (cross fingers) that by the time we release the class hierarchy will be simplified from (Entry -> Tool, (Workflow -> BioWorkflow, Service)) to (Entry -> Tool, Workflow, Service) 

We just sucked at coming up with names for "thing that unifies workflows and services but not tools because we haven't gotten that far yet"